mobile/test/common/integration/test_server_interface.h

```diff
@@ -25,6 +25,12 @@ void shutdown_server();
  */
 int get_server_port();
 
+/**
+ * Set dummy response data for server. Can only be called once the server has been started.
```

Comment:
nit: Let's remove "dummy" from this comment. This is the actual headers and data which will be returned. Does it return this header+body for any URL? If so, please mention.

Reply:
Updated, and yes it's for any URL